flux/model.py

```diff
@@ -33,3 +36,30 @@ def compute_steady_state_temp(FF, E, rho, emiss,
         return np.array(T)
     else:
         raise Exception('E should be a vector or a 2D array')
+
+def update_incoming_radiances(FF, E, rho, emiss, Qrefl, QIR, Tsurf, clamp=True):
+    """
+    Compute the fluxes/irradiances at time step
+    Args:
+        FF: form factor matrix (constant,
+        E: direct flux at time t
+        rho: albedo
+        emiss: emissivity
+        Qrefl: reflected flux at previous time step
+        QIR: IR flux at previous time step
+        Tsurf: surface temperature at previous time step
+        clamp: boolean
+
+    Returns:
+        Qrefl_np1, QIR_np1
+    """
+    # eq 17 radiosity paper
+    Qrefl_np1 = rho * FF @ (E + Qrefl)
```

Comment:
Is this ok? I saw that the `solve_radiosity` function has a `albedo_placement` option ...

Reply:
Also, I'm not sure why Eq. 17 uses `A_{n+1} = \rho_{n+1}`: I suppose sometimes albedo can change over time but here it is constant, so this should be ok.

Reply:
On Mars there is seasonal frost, which is much brighter, so at some point one needs time-dependent albedo.

Reply:
I believe the placement of the rho is correct, but our notation is very confusing. In the radiosity equation (and our paper), rho is left of the sum, whereas the albedo (called rho in the program) is inside the sum. In `solve_radiosity` rho can be left or right of the sum, and the default choice is `albedo_placement=right`, opposite to the convention in the radiosity requation. In the program, maybe we should just use `albedo` for albedo instead of `rho`.

Reply:
We could maybe rename `rho` to `albedo` in `lsp_spice.py` but leave it as `rho` in the `flux/model.py` functions arguments? I'm not sure where and when it would make sense to keep it as `rho` (and yes, I was also a bit confused at the beginning among `A`, `rho`, `albedo`... :) ).